src/initializers/datadog/index.ts

```diff
@@ -1,5 +1,6 @@
 import requireInjected from '../../require-injected';
 import { getLogger } from '../log4js';
+import { formats } from 'dd-trace/ext';
```

Comment:
@klesgidis dd-trace might not be available (if the project is not using dd). I think this will throw an error in such cases